app/connectors/utilities/ConnectorUtils.scala

```diff
@@ -24,17 +24,17 @@ trait ConnectorUtils[L <: ErrorResponsesModel, R] extends JsonUtils {
 
   type Response = Either[ErrorModel, R]
 
-  def parseFailure(status: Int, jsValue: JsValue)(implicit lReader: Reads[L]): Response = {
+  def parseFailure(status: Int, jsValue: JsValue)(implicit lReader: Reads[L]): Response = Left (
```

Comment:
I am not sure about this part, why did you add the `Left`?

Reply:
It was always there - it was just added magically by an implicit.  I have removed the implicit because it really was making the codebase more complex, not less (and the implicit file wasn't well covered, which is the original point of the ticket).